src/sdkModels.ts

```diff
@@ -179,21 +246,20 @@ export interface IMethodResponse {
   mediaType: string
   type: IType
   mode: ResponseMode
+  description: string
 
   search(rx: RegExp, criteria: SearchCriteria): boolean
 
   searchString(criteria: SearchCriteria): string
 }
 
 class MethodResponse implements IMethodResponse {
-  mediaType: string
-  statusCode: number
-  type: IType
 
-  constructor(statusCode: string, mediaType: string, type: IType) {
-    this.statusCode = parseInt(statusCode, 10)
-    this.mediaType = mediaType
-    this.type = type
+  constructor(
+    public statusCode: number,
+    public mediaType: string,
+    public type: IType,
+    public description: string) {
```

Comment:
slick!

Reply:
Are you talking about Typescript's syntactic sugar to declare a constructor parameter as a public property?

Reply:
that and the auto-assignment to those public members at instantiation time. nice to remove boilerplate